Firestore/src/V1/Gapic/FirestoreGapicClient.php

```diff
@@ -605,57 +418,55 @@ public function beginTransaction($database, array $optionalArgs = [])
      * ```
      * $firestoreClient = new FirestoreClient();
      * try {
-     *     $formattedDatabase = $firestoreClient->databaseRootName('[PROJECT]', '[DATABASE]');
-     *     $writes = [];
-     *     $response = $firestoreClient->commit($formattedDatabase, $writes);
+     *     $response = $firestoreClient->commit();
      * } finally {
      *     $firestoreClient->close();
      * }
      * ```
      *
-     * @param string  $database Required. The database name. In the format:
-     *                          `projects/{project_id}/databases/{database_id}`.
-     * @param Write[] $writes   The writes to apply.
-     *
-     * Always executed atomically and in order.
      * @param array $optionalArgs {
-     *                            Optional.
+     *     Optional.
      *
+     *     @type string $database
+     *           Required. The database name. In the format:
+     *           `projects/{project_id}/databases/{database_id}`.
+     *     @type Write[] $writes
+     *           The writes to apply.
+     *
+     *           Always executed atomically and in order.
      *     @type string $transaction
-     *          If set, applies all writes in this transaction, and commits it.
+     *           If set, applies all writes in this transaction, and commits it.
      *     @type RetrySettings|array $retrySettings
-     *          Retry settings to use for this call. Can be a
-     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
-     *          of retry settings parameters. See the documentation on
-     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     *           Retry settings to use for this call. Can be a
+     *           {@see Google\ApiCore\RetrySettings} object, or an associative array of retry
+     *           settings parameters. See the documentation on
+     *           {@see Google\ApiCore\RetrySettings} for example usage.
      * }
      *
      * @return \Google\Cloud\Firestore\V1\CommitResponse
      *
      * @throws ApiException if the remote call fails
      */
-    public function commit($database, $writes, array $optionalArgs = [])
+    public function commit(array $optionalArgs = [])
```

Comment:
This breaks BC and needs to be fixed. It's strange because "database" is currently an [optional-to-required fix](https://github.com/googleapis/gapic-generator-php/blob/main/src/Generation/FieldDetails.php#L121), which implies this should have already been optional, so we need to consult the protos on this one.

Reply:
FIxed in https://github.com/googleapis/gapic-generator-php/pull/451